app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,8 @@
  *= require_tree .
  *= require_self
  */
+
+ @import "tailwindcss/base";
+ @import "tailwindcss/components";
+ @import "tailwindcss/utilities";
+ 
```

Comment:
The line break mark still appears.

Will you have an extra space?

![imagen](https://user-images.githubusercontent.com/12243331/101553593-9267c680-397a-11eb-9882-049a75324b6a.png)

Reply:
What about now, I added a new line with `echo`, the file looks fine now, what do you think?